libevmcore/Instruction.cpp

```diff
@@ -82,6 +82,8 @@ const std::map<std::string, Instruction> dev::eth::c_instructions =
 	{ "SSTORE", Instruction::SSTORE },
 	{ "JUMP", Instruction::JUMP },
 	{ "JUMPI", Instruction::JUMPI },
+	{ "JUMPV", Instruction::JUMP },
```

Comment:
What about prefixing them? Oh, now I see that it actually says `Instruction::JUMP`, is that intended?